File: Selenium/Project/CRM_Activity6.java
package project;

import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.interactions.Actions;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class CRM_Activity6 {
	
	//declare driver
			WebDriver driver;
		
			
			@BeforeClass
			  public void beforeClass() {
				
				//Initializing
				  driver = new ChromeDriver();
				  
			  }
				
  @Test
  public void verifyActivityMenu() {
	  
	  //Login
	 CRM_Activity4 login = new CRM_Activity4();
	 login.verifyLogin(driver);
	  
	  
	  //Verify Activity menu exists
	 String activityText = driver.findElement(By.xpath("*//a[text()='Activities']")).getText();
	 Assert.assertEquals(activityText, "ACTIVITIES");
  }
  
  @AfterClass
  public void afterClass() {
	  
	  driver.quit();
  }
  
}

File: Selenium/Project/CRM_Activity6.py
# Import 
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC
from CRM_Activity4 import verify_login

# Start the Driver
with webdriver.Chrome() as driver:
    # Navigate to the URL
        driver.get("https://alchemy.hguy.co/crm/")

        #Login
        verify_login(driver)

        #Verify Activity Menu
        activityText = driver.find_element(By.XPATH,"*//a[text()='Activities']").text
        assert activityText=="ACTIVITIES", "Text does not match!"
        
driver.quit()